app/views/open_data/show.html.erb

```diff
@@ -97,6 +97,12 @@
       <%= render partial: 'tables', object: tables %>
       <%= render partial: 'table', collection: tables %>
 
+      <h3 id="ra">Register adries <span class="label label-danger">ALFA</span></h3>
+      <p>Zmenové dáta Registra adries</p>
```

Comment:
V ich [dokumentacii](https://data.gov.sk/dataset/de3dd18f-9124-4acb-ae00-705555332256/resource/3513f28f-0ced-45fb-b1b9-be31aacb999a/download/package1doc1.htm) je to vsade s velkym R, ale napr na [minv.sk](http://www.minv.sk/?register-adries) je to zase 'register'. Prerobim.